helloscala-lagom-client/src/main/resources/application.conf

```diff
@@ -0,0 +1 @@
+play.server.netty.log.wire = on
```

Comment:
This is really not necessary but even adding an `application.conf`  would not be enough to workaround https://github.com/lagom/lagom/issues/412